source/docs/hardware/hardware-basics/wiring-pneumatics.rst

```diff
@@ -14,54 +14,29 @@ roboRIO over the CAN bus and powered via 12V from the PDP.
-anywhere in the middle of the CAN chain.
+The first PCM on your robot can be wired from the PDP VRM/PCM connectors on the end of the PDP. The PCM is connected to the roboRIO via CAN and can be placed anywhere in the middle of the CAN chain (or on the end with a custom terminator). For more details on wiring a single PCM see :ref:`Wiring the FRC Control System <docs/getting-started/getting-started-frc-control-system/how-to-wire-a-robot:Pneumatics Control Module Power (Optional)>`. Additional PCMs can be wired to a standard WAGO connector on the side of the PDP and protected with a 20A or smaller circuit breaker. Additional PCMs should also be placed anywhere in the middle of the CAN chain.
 
 The Compressor
 ---------------
 
-The compressor can be wired directly to the Compressor Out
-connectors on the PCM. If additional length is required,
-make sure to use 18 AWG wire or larger for the extension.
+The compressor can be wired directly to the Compressor Out connectors on the PCM. If additional length is required, make sure to use 18 AWG wire or larger for the extension.
 
 The Pressure Switch
 ----------------------------
 
-The pressure switch should be connected directly to
-the pressure switch input terminals on the PCM. There
-is no polarity on the input terminals or on the pressure
-switch itself, either terminal on the PCM can be connected
-to either terminal on the switch. Ring or spade terminals
-are recommended for the connection to the switch screws
-(note that the screws are slightly larger than #6, but can
-be threaded through a ring terminal with a hole for a #6
-screw such as the terminals shown in the image).
+The pressure switch should be connected directly to the pressure switch input terminals on the PCM. There is no polarity on the input terminals or on the pressure switch itself, either terminal on the PCM can be connected to either terminal on the switch. Ring or spade terminals are recommended for the connection to the switch screws (note that the screws are slightly larger than #6, but can be threaded through a ring terminal with a hole for a #6 screw such as the terminals shown in the image).
 
 Solenoids
 ---------
 
-Each solenoid channel should be wired directly to a
-numbered pair of terminals on the PCM. A single
-acting solenoid will use one numbered terminal pair.
-A double acting solenoid will use two pairs (as shown
-in the image above). If your solenoid does not come
-with color coded wiring, check the datasheet to make
-sure to wire with the proper polarity.
+Each solenoid channel should be wired directly to a numbered pair of terminals on the PCM. A single acting solenoid will use one numbered terminal pair. A double acting solenoid will use two pairs (as shown in the image above). If your solenoid does not come with color coded wiring, check the datasheet to make sure to wire with the proper polarity.
 
 Solenoid Voltage Jumper
 ------------------------
 
 .. image:: images/wiring-pneumatics/pcm01.jpg
```

Comment:
Can we find a higher resolution source image? Removing the width (still wanted) reduces the visual size. I think we should specify a source image minimum width would be good in the style guide (I'll create an issue for it).

Reply:
Does the illustration I added in 83dbaad work well?